src/components/v5/common/ActionSidebar/partials/ActionSidebarContent/ActionSidebarContent.tsx

```diff
@@ -306,6 +306,11 @@ const ActionSidebarContent: FC<ActionSidebarContentProps> = ({
                 include: [SearchActionsDocument],
               });
             }
+            if (isQueryActive('SearchStreamingPayments')) {
```

Comment:
would it be possible to do this invalidation somewhere else more streaming payment related? like its form

Reply:
Moved to completed action component